prototype-implementation/e2e-test/testData/projects/settings/module.yaml

```diff
@@ -54,3 +54,9 @@ settings:
       - -Xinline-classes
     optIns:
       - kotlinx.Experimental
+  kover:
+    enabled: true
+    html:
+      onCheck: true
```

Comment:
When we enable kover, it creates tasks to explicitly run the various reports. For example, with `enabled: true`, I can run `./gradlew koverHtmlReport`, and it will generate an HTML report. By default, kover will enable all the reports like this, but there's a way to fully disable specific reports. It uses a different project extension, though, and since the report types are only run explicitly by default, I didn't add a disabled variable for each one.

The `onCheck` setting tells kover to make the `check` task depend on that report. If I don't care about the log report, for example, I may not want it to run when I `check` my project, but I may want an HTML report generated. The most common one to want to set `onCheck: true` for is likely the verify task, which I'll add configuration for when I get the chance.

Reply:
If the goal is to make things work with minimal config, I can change the defaults. Would it be good to have `kover: enabled: true` by default, and perhaps set `onCheck: true` for html or xml? I'd be hesitant to enable verify by default, since that task is meant to ensure minimum coverage via different metrics, which is often a rule set by individual teams. If y'all do want it enabled by default, it would be good to know what the defaults should be when I write the config for it.

Reply:
Thanks for explaining. Feel free to use your own judgement to set the defaults. The guides that could help you: if I don't use kover, there should not be performance penalty. If I want to use the default workflow, simply `kover:enabled:true` should be enough.